volttrontesting/utils/platformwrapper.py

```diff
@@ -1507,25 +1521,23 @@ def create_web_admin(self, username, password, messagebus='rmq'):
         :param password:
         :return:
         """
-        #     from volttron.platform.web.admin_endpoints import AdminEndpoints
-        #     from volttrontesting.utils.web_utils import get_test_web_env
-        #
-        #     params = urlencode(dict(username='admin', password1='admin', password2='admin'))
-        #     env = get_test_web_env("/admin/setpassword", method='POST')  # , input_data=input)
-        #     adminep = AdminEndpoints()
-        #     resp = adminep.admin(env, params)
-        # else:
-        if messagebus == 'rmq':
-            data = dict(username=username, password1=password, password2=password)
-            url = self.bind_web_address + "/admin/setpassword"
-            # resp = requests.post(url, data=data,
-            # verify=self.certsobj.remote_cert_bundle_file())
-            resp = requests.post(url, data=data,
-                                 verify=self.certsobj.cert_file(
-                                     name=self.certsobj.root_ca_name))
-            return resp
-        else:
-            return None
+        from volttron.platform.web.admin_endpoints import AdminEndpoints
+        from volttrontesting.utils.web_utils import get_test_web_env
+
+        # params = urlencode(dict(username='admin', password1='admin', password2='admin'))
+        # env = get_test_web_env("/admin/setpassword", method='POST')  # , input_data=input)
+        # adminep = AdminEndpoints()
+        # resp = adminep.admin(env, params)
+        # # else:
+        data = dict(username=username, password1=password, password2=password)
+        url = self.bind_web_address + "/admin/setpassword"
+        # resp = requests.post(url, data=data,
+        # verify=self.certsobj.remote_cert_bundle_file())
```

Comment:
how does it work in case of zmq with http

Reply:
Modified to check if ssl enabled or not. Will not use root ca to verify request if ssl is not enabled.